ui/raidboss/data/00-misc/general.ts

```diff
@@ -211,7 +215,8 @@ const triggerSet: TriggerSet<Data> = {
         'has initiated a ready check': 'eine Bereitschaftsanfrage gestellt',
         'You have commenced a ready check': 'Du hast eine Bereitschaftsanfrage gestellt',
         'You poke the striking dummy': 'Du stupst die Trainingspuppe an',
-        'You psych yourself up alongside the striking dummy': 'Du willst wahren Kampfgeist in der Trainingspuppe entfachen',
```

Comment:
dprint doesn't have a config for this, for me I really like this didn't break into new line

Reply:
Yeah, I think it's not great, but I also don't want lines to get longer in non-trigger files either.  I think the fact that it auto-formats and this only tends to happen on really long game log lines infrequently makes it not as bad as it could be.